core/src/main/scala/fortress/config/Manager.scala

```diff
@@ -0,0 +1,102 @@
+package fortress.config
+
+import fortress.compiler._
+import fortress.modelfind._
+import fortress.operations._
+import fortress.config._
+import fortress.util._
+
+/**
+  * Used to create a compiler and modelfinder in combitionation.
+  *
+  */
+class Manager {
+    
+    // The options
+    protected val options: collection.mutable.Map[String, ConfigOption] = new collection.mutable.HashMap[String, ConfigOption]()
+
+    // The priority given for each option
```

Comment:
If priority = sequence, let's rename it.

Reply:
Updated